doc/rtd/topics/testing.rst

```diff
@@ -36,6 +36,16 @@ Test Layout
     subclass (indirectly) from ``TestCase`` (e.g.
     `TestPrependBaseCommands`_)
 
+* Unit tests and integration tests are located under cloud-init/tests
```

Comment:
Good addition, but lets remove the 'found in-tree alongside the source code' reference on line 6 too.